deployment/aws/infra_configs/cluster_features.sh

```diff
@@ -1,14 +0,0 @@
-# privateAccess enable private access for your Amazon EKS cluster's Kubernetes API server endpoint
```

Comment:
Hi @Jeffwan - thanks for your work on this. Did you delete this intentionally? For v0.6.1 this broke https://github.com/kubeflow/kubeflow/blob/master/scripts/aws/util.sh - which relies on cluster_features.sh

Following https://www.kubeflow.org/docs/aws/deploy/install-kubeflow/ with export KUBEFLOW_TAG=v0.6.1

Reply:
Oh. I see. Thanks for letting me know. This will be only used in 0.5.1 and 0.6 has few changes for this. I will update docs this week to give latest guidance.

Reply:
Cool - thanks!